glass/src/libnt/native/include/glass/networktables/NetworkTables.h

```diff
@@ -88,17 +88,19 @@ class NetworkTablesModel : public Model {
 
 using NetworkTablesFlags = int;
 
+static constexpr const int kPrecisionBitShift = 6;
```

Comment:
Since this is outside a class, it should have a more precise name, eg kNetworkTablesFlags_PrecisionBitShift.